Library/Homebrew/test.rb

```diff
@@ -25,7 +25,9 @@
 
   # tests can also return false to indicate failure
   Timeout.timeout TEST_TIMEOUT_SECONDS do
-    raise "test returned false" if formula.run_test == false
+    formula.with_logging("test") do
+      raise "test returned false" if formula.run_test == false
+    end
```

Comment:
They're redundant. Doesn't really matter which one, since they guard the same potential `system` calls. Removing this one in favor of the one inside `run_test`, for consistency with `run_postinstall`, and because it's possible `run_test` could get called from elsewhere, and it should probably establish that logging context regardless of the caller.